src/app/dynamic-forms/fields/rich-text/rich-text.component.ts

```diff
@@ -156,22 +179,51 @@ export class RichTextComponent implements AfterViewInit, OnDestroy {
   ngAfterViewInit(): void {
 
     let content: string | Object;
+    const self = this
 
     try {
       content = ((this.field.value as string).includes(`"type":"doc"`)) ? JSON.parse(this.field.value ?? "{}") : this.field.value;
     }
     catch (error) {
       content = this.field.value;
     }
-
+    
     if (this.canEdit) {
+      // Disable spellcheck in code blocks
+      const CustomCodeBlockLowlight = CodeBlockLowlight.extend({
+        addAttributes() {
+          return {
+            spellcheck: { default: 'false' },
+          }
+        }
+      });
+
       this.editor = new Editor({
         element: this.richTextElement?.nativeElement,
         extensions: [
-          StarterKit,
+          CustomCodeBlockLowlight.configure({
+            lowlight,
+          }),
           Link.extend({
+            addKeyboardShortcuts() {
+              return {
+                'Mod-k': () => {
+                  self.toggleLink()
+                  return true                
+                },
+                'Ctrl-k': () => {
```

Comment:
Using `Mod` is a shorthand for `Cmd` on Mac or `Ctrl` otherwise, so this second block is unnecessary.